ci/arm/gitlab-ci.yml

```diff
@@ -0,0 +1,10 @@
+linux-arm64:
+  script:
+    - uname -a
+    - lscpu
+    - free -h
+    - mkdir build
+    - cd build
+    - PATH=/usr/lib/ccache:$PATH cmake .. -DCMAKE_BUILD_TYPE=Release -GNinja
```

Comment:
You assume ccache is installed?

Reply:
this doesn't assume that, but will use it if it is installed